example/snippets.cpp

```diff
@@ -0,0 +1,805 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CppAlliance/url
```

Comment:
The capitalization is not quite right on the URL, which might cause it to be skipped when we do a find and replace.